girara/session.c

```diff
@@ -403,8 +404,10 @@ bool girara_session_init(girara_session_t* session, const char* sessionname) {
 
   /* notification area */
   gtk_container_add(GTK_CONTAINER(session->gtk.notification_area), session->gtk.notification_text);
-  gtk_widget_set_halign(session->gtk.notification_text, GTK_ALIGN_START);
+  gtk_label_set_xalign(GTK_LABEL(session->gtk.notification_text), 0.0);
   gtk_widget_set_valign(session->gtk.notification_text, GTK_ALIGN_CENTER);
+  gtk_widget_set_margin_start(session->gtk.notification_text, 2);
+  gtk_widget_set_margin_end(session->gtk.notification_text, 2);
```

Comment:
This should go into the css file and be configurable (ideally).

Reply:
I will just probably just drop it, since there is already `statusbar-h-padding` and `statusbar-v-padding`.